libs/header/sync/sync.go

```diff
@@ -247,9 +240,8 @@ func (s *Syncer[H]) doSync(ctx context.Context, fromHead, toHead H) (err error)
 	return err
 }
 
-// processHeaders gets and stores headers starting at the given 'from' height up to 'to' height -
-// [from:to]
-// processHeaders checks headers in pending cache that apply to the requested range.
+// processHeaders fetches and stores asked headers [from:to].
```

Comment:
for now, the range is (from; to]

Reply:
Ok, I just kept it like it was before, but good catch

Reply:
Fixed